pages/android/getprop.md

```diff
@@ -1,32 +1,32 @@
-# adb shell getprop
+# getprop
 
 > Show information about adb devices and emulators.
 > More information: <https://adbinstaller.com/commands>.
```

Comment:
This URL really isn't very specific. Is there something better we can reference?

Reply:
@sbrl https://github.com/tldr-pages/tldr/pull/5457#discussion_r595099178

Reply:
As I mentioned in the other PR, I'm not convinced of the usefulness of the generic link here.

Reply:
I found [https://adbshell.com/commands/adb-shell-getprop](https://adbshell.com/commands/adb-shell-getprop), but the commands all start with `adb shell`.

Reply:
Hrm, not a big fan of that either really sorry.